Dockerfile-management

```diff
@@ -0,0 +1,10 @@
+FROM ubuntu:22.04
```

Comment:
Isn't it way to early?

Reply:
it has osm2pgsql 1.6 which only works for me, no luck with the oldest versions in prev dist versions

Reply:
Fair enough,